dtfs-central-api/src/utils/getObjectPropertyValueFromStringPath.js

```diff
@@ -1,9 +1,7 @@
-const getObjectPropertyValueFromStringPath = (obj, str) =>
-  str.split('.').reduce((p, propertyName) => {
-    if (p && p[propertyName]) {
-      return p[propertyName];
-    }
+const getObjectPropertyValueFromStringPath = (deal, field) =>
+  field.split('.').reduce((o, p) => {
+    if (o && o[p]) return o[p];
     return null;
-  }, obj);
+  }, deal);
```

Comment:
I'd keep this as obj because then it's generic, could be used on things that aren't deals 😉